web/utils.go

```diff
@@ -34,6 +37,21 @@ func Userget(r *http.Request) (models.User, error) {
 	return user, nil
 }
 
+func encrypt(word string) (string, error) {
```

Comment:
Got you.

btw: I just realized you are critical about some theoretical computer science topics (Big O). thats cool.